docs/siteconfig.rst

```diff
@@ -64,9 +64,6 @@ Here is a sample configuration with many of the options set and documented::
     # Where the ceph git repo is considered to reside.
     ceph_git_url: https://github.com/ceph/ceph.git
 
-    # Where the ceph-qa-suite git repo is considered to reside.
```

Comment:
keep this: it still is a valid configuration option, only we don't want teuthology-openstack to set it by default